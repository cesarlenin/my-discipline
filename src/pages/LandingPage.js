import React, { Component } from 'react'
import LogInform from '../components/LogInform/LogInform'

export class LandingPage extends Component {
  render() {
    return (
    <div className='landingPage'>
        <header className='landingPage_header'>
            <h1>MyDiscipline</h1>
            <h3>Be Organized Be Discipline</h3>
        </header>
        <section>
        <LogInform/>
        </section>
    </div>
    )
  }
}

export default LandingPage