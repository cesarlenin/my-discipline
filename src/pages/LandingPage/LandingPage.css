.smallerH1{
    font-size: 23px;
}

.landingPage_header{
    padding: 35px 15px 0px 15px;
}

.log-in{
    padding: 20px 0px;
}

.appInfo{
    margin: auto;
    font-size: 14px;
}
