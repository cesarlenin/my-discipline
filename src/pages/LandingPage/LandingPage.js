import React, { Component } from 'react';
import LogInform from '../../components/LogInform/LogInform';
import './LandingPage.css';
export class LandingPage extends Component {
  render() {
    return ( 
      <section className = 'landingPage' >
        <header className = 'landingPage_header' >
          <h1 className = 'smallerH1' > Be Organized< br />
          Be Disciplined < br/>
          Be A Better You </h1> 
        </header> 
        <section className = 'landingContainer' >
          <p className = 'appInfo' > My Discipline helps you keep track of your habits.< br />
          Start by setting goals you want to meet.< br />
          Then build a collection of habits you want to have,< br />
          and see their actual frequency in a calendar. 
          </p> 
          <section className = 'log-in' >
            <LogInform />
          </section> 
        </section> 
      </section>
    )
  }
}

export default LandingPage;