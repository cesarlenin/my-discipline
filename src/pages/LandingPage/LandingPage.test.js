import React from 'react';
import ReactDOM from 'react-dom';
import LandingPage from './LandingPage';
import renderer from 'react-test-renderer';

describe('<LandingPage />', () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render( <
      LandingPage / > , div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders this UI as expected', () => {
    const tree = renderer.create( < LandingPage /> ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});