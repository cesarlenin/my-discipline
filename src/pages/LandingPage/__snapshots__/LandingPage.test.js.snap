// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPage /> renders this UI as expected 1`] = `
<section
  className="landingPage"
>
  <header
    className="landingPage_header"
  >
    <h1
      className="smallerH1"
    >
      Be Organized
      <br />
      Be Disciplined
      <br />
      Be A Better You
    </h1>
  </header>
  <section
    className="landingContainer"
  >
    <p
      className="appInfo"
    >
      My Discipline helps you keep track of your habits.
      <br />
      Start by setting goals you want to meet.
      <br />
      Then build a collection of habits you want to have,
      <br />
      and see their actual frequency in a calendar.
    </p>
    <section
      className="log-in"
    >
      <form
        className="LoginForm"
        onSubmit={[Function]}
      >
        <div
          role="alert"
        />
        <div
          className="user_name"
        >
          <label
            htmlFor="LoginForm__user_name"
          >
            User name:
          </label>
          <input
            className="Input "
            id="LoginForm__user_name"
            name="user_name"
            required={true}
          />
        </div>
        <div
          className="password"
        >
          <label
            htmlFor="LoginForm__password"
          >
            Password:
          </label>
          <input
            className="Input "
            id="LoginForm__password"
            name="password"
            required={true}
            type="password"
          />
        </div>
        <button
          className="Button "
          type="submit"
        >
          Login
        </button>
      </form>
    </section>
  </section>
</section>
`;
